Source/Utilities.H

```diff
@@ -199,7 +261,8 @@ pc_ctoprim(
   amrex::Array4<const amrex::Real> const& u,
   amrex::Array4<amrex::Real> const& q,
   amrex::Array4<amrex::Real> const& qa,
-  PassMap const& pmap)
+  PassMap const& pmap,
+  const int clean_massfrac = 1)
```

Comment:
Why is this set to 1 here?

Reply:
being consistent with our default from pelec_queries. see https://github.com/AMReX-Combustion/PeleC/blob/ba4b807d8cf13b5a691debc49a9ca1da6533abf8/Source/Params/param_includes/pelec_defaults.H#L105

Reply:
I think we should remove the `=1` since the default or whatever is set should be passed in right? I think it needs to be added to https://github.com/AMReX-Combustion/PeleC/blob/ba4b807d8cf13b5a691debc49a9ca1da6533abf8/Source/LES.cpp#L467 as well.

Reply:
sounds good. I have made the change.

Reply:
Awesome, thanks.